### DISCUSSION TITLE
Questions about the education certification of Chinese students

### DISCUSSION THREAD

**Discussion by xhemj** [community/community#52728] [GitHub Education] [answered]
I am a student in China and I applied to the school for a "Certificate of Attendance" with the school's stamp in order to have it validated.

However, as the original certificate is in Chinese, it keeps showing that the name of the school is not recognised when I upload it (the name is listed in English), and when I manually translate it on the photo and upload it again, it says that the photo has been changed and cannot be verified.

What can I do please?

**Comment by pnpn111** [comment, marked as answer]
It is simple to write the English translation next to the material and then upload the original text with the translation by taking a photo.For more, 
you check out [my article](https://www.luogu.com.cn/blog/pnpn111/github-student-packet).

**Reply by whitewatercn** [reply]
> It is simple to write the English translation next to the material and then upload the original text with the translation by taking a photo.For more, you check out [my article](https://www.luogu.com.cn/blog/pnpn111/github-student-packet).

感谢大佬！真滴有用，在此基础上我做了一些补充 https://xlog.whitewater.wang/Github_Student_Developer_Pack
this is useful! and i make some supplyment based on your post https://xlog.whitewater.wang/Github_Student_Developer_Pack

**Reply by Ntshangase** [reply]
Hey @whitewatercn  you links are broken.  Error : `This site can't be reached`
